odoo/addons/base/wizard/base_language_install_views.xml

```diff
@@ -9,7 +9,7 @@
                 <form string="Load a Translation">
                     <field name="state" invisible="1"/>
                     <group states="init">
-                        <field name="lang"/>
+                        <field name="lang" widget="many2many_tags"/>
```

Comment:
Probably a commit rebase / split mismatch, in first commit about crm there is a m2m tags added on lang in base language install. It should probably be done in second commit.